packages/node/__tests__/lib/construct-payload.test.ts

```diff
@@ -70,14 +67,15 @@ describe('constructPayload()', () => {
   });
 
   it('should still support id even though it has been deprecated in favor of apiKey', () => {
-    const group = {
+    const groupAlt = {
```

Comment:
`group` is already defined in the `describe` that this test is contained within, but this test seems odd because it's saying it should support `id` but we aren't passing `id in here at all. Bad test?

Reply:
yeah looks like a bad test